debian/changelog

```diff
@@ -1,3 +1,11 @@
+daos (2.2.0-9) unstable; urgency=medium
+  [ Michael Hennecke ]
+  * Change ipmctl requirement from v2 to v3
+  * Change libfabric requirement from 1.14.0-1 to 1.15.1-1
+  * Change spdk requirement from 22.01.1 to 22.01.2
+
+ -- Michael Hennecke <michael.hennecke@intel.com> Fri, 10 Feb 2023 16:40:00 -0100
+
```

Comment:
Since it seems to have gotten missed by yours truly, could you add the missing 2.2.0-8 entry here:

```suggestion

daos (2.2.0-8) unstable; urgency=medium
  [ Brian J. Murrell ]
  * NOOP change to keep in parity with RPM version

 -- Brian J. Murrell <brian.murrell@intel.com>  Fri, 3 Feb 2023 00:00:00 -0000

```

Please and thanks!